scripts/jest/setupTests.js

```diff
@@ -205,8 +205,17 @@ if (process.env.REACT_CLASS_EQUIVALENCE_TEST) {
     }
   };
 
+  const coerceGateConditionToFunction = gateFnOrString => {
+    return typeof gateFnOrString === 'string'
```

Comment:
Unrelated to this improvement, it could be nice to assert a boolean result of the function to fail earlier.

Reply:
Was thinking the same. Would help clean up removed flags.

Reply:
> Would help clean up removed flags.

there's already an error if you reference a flag that doesn't exist